xcm/src/v3/traits.rs

```diff
@@ -157,6 +157,8 @@ pub enum Error {
 	WeightNotComputable,
 	/// Recursion stack limit reached
 	ExceedsStackLimit,
+	/// Conversion from XCVM Assets to MultiAssets might fail because of too many assets
+	TooManyAssets,
```

Comment:
This looks like it's an error that needs to be promoted to a properly indexed one. We'll need to change the spec to reflect this.